logging/event_codes.go

```diff
@@ -46,7 +46,10 @@ const (
 	EventCodeErrorCantParseRequestData    = 552
 	EventCodeErrorCantGetCurrentConfig    = 553
 	EventCodeErrorCantSetNewConfig        = 554
-	EventCodeErrorCantDumpConfig          = 555
+	EventCodeErrorCantHashPassword        = 555
+	EventCodeErrorCantGetAuthData         = 556
+	EventCodeErrorCantParseAuthData       = 557
+	EventCodeErrorCantDumpConfig          = 558
```

Comment:
after release 0.77 we should change error codes really carefully, because people will use them

Reply:
Sure, I thought exactly the same but solved with min effort)